container/src/constants/communication.ts

```diff
@@ -1,176 +1,413 @@
+   * }
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const ALERT_REQUEST = 'show-alert-request';
 
   /**
    * Event left due to historical reasons - do not use.
    * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const ALERT_CLOSED = 'close-alert-request';
 
   /**
    * Event fired when the micro frontend has been initialized.
+   * It's dispatched with custom data.
+   * @type {Object}
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const INITIALIZED = 'initialized';
 
   /**
    * Event fired when the micro frontend requests the addition of search parameters to the URL.
+   * It's dispatched with custom data shown below (check {@link https://github.com/SAP/luigi/blob/main/container/typings/constants/event-payloads.ts|ParamsRequestPayload} details) - some properties are optional.
+   * @type {ParamsRequestPayload}
+   * @example
+   * {
+   *  data: {},
+   *  keepBrowserHistory: false
+   * }
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const ADD_SEARCH_PARAMS_REQUEST = 'add-search-params-request';
 
   /**
    * Event fired when the micro frontend requests the addition of node parameters to the URL.
+   * It's dispatched with custom data shown below (check {@link https://github.com/SAP/luigi/blob/main/container/typings/constants/event-payloads.ts|ParamsRequestPayload} details) - some properties are optional.
+   * @type {ParamsRequestPayload}
+   * @example
+   * {
+   *  data: {},
+   *  keepBrowserHistory: false
+   * }
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const ADD_NODE_PARAMS_REQUEST = 'add-node-params-request';
 
   /**
    * Event fired when the micro frontend requests to show a confirmation modal.
+   * It's dispatched with custom data and the payload shown below - some properties are optional. Read more about `showConfirmationModal` params [here](https://docs.luigi-project.io/docs/luigi-core-api?section=showconfirmationmodal).
+   * @type {Object.<string, unknown | ConfirmationModalRequestPayload>}
+   * @property {ConfirmationModalRequestPayload} payload - event payload (check {@link https://github.com/SAP/luigi/blob/main/container/typings/constants/event-payloads.ts|ConfirmationModalRequestPayload} details)
+   * @example
+   * payload: {
+   *  header: 'Confirmation',
+   *  body: 'Are you sure you want to do this?',
+   *  buttonConfirm: 'Yes',
+   *  buttonDismiss: 'No'
+   * }
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const SHOW_CONFIRMATION_MODAL_REQUEST = 'show-confirmation-modal-request';
 
   /**
    * Event fired when the micro frontend requests to show a loading indicator.
+   * It's dispatched with custom data.
+   * @type {Object}
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const SHOW_LOADING_INDICATOR_REQUEST = 'show-loading-indicator-request';
 
   /**
    * Event fired when the micro frontend requests to hide the loading indicator.
+   * It's dispatched with custom data.
+   * @type {Object}
```

Comment:
Since we don't offer any payload which can be used we should not mention it in our docu.